JavascriptMornings/Maitee
# Add display of current Fuel

Add a bar with the current amount of gas that the car has
## Review checklist


## Testing

Steps to test the change:
change the `currentGallons` variable inside car.js and see if the fuel bar changes accordingly

<img width="742" alt="screen shot 2018-03-22 at 11 24 08 am" src="https://user-images.githubusercontent.com/29434543/37780776-d5ec9b9c-2dc5-11e8-8f92-9caa47e0cbea.png">

## Additional notes

## Deploy checklist
